graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH7.kt

```diff
@@ -0,0 +1,136 @@
+package com.anaplan.engineering.azuki.graphs.eacs
+
+import com.anaplan.engineering.azuki.core.runner.Eac
+import com.anaplan.engineering.azuki.core.system.BEH
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphBehaviours
+import com.anaplan.engineering.azuki.graphs.adapter.api.GraphFunctions
+import com.anaplan.engineering.azuki.graphs.dsl.GraphScenario
+import com.anaplan.engineering.azuki.graphs.graphA
+
+@BEH(GraphBehaviours.GetSimpleCycleCount, GraphFunctions.GetSimpleCycleCount, """
+    Determine the number of cycles in a given directed graph
+""")
+class BEH7 : GraphScenario() {
+
+    @Eac("A cycle is a continuous path which starts and ends at the same vertex")
```

Comment:
🚫 We should define what a _simple_ cycle is, rather than a plain cycle, since that is what this function counts.